.circleci/setup-android-on-ubuntu.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
I put this stuff in a dedicated script to keep the pipeline cleaner and to make it easier to reuse it in the future.